src/Aspire.ProjectTemplates/templates/aspire-mstest/9.1/Aspire.Tests.1.csproj

```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Aspire.Hosting.Testing" Version="!!REPLACE_WITH_LATEST_VERSION!!" />
+    <PackageReference Include="Aspire.Hosting.Testing" Version="9.1.0" />
```

Comment:
Should this become 9.1.1 - aren't we doing a servicing release or are we going to wait for 9.2?

Reply:
When we ship 9.1.1, we can update this 😄